lib/services/systemd.c

```diff
@@ -171,7 +203,7 @@ systemd_loadunit_result(DBusMessage *reply, svc_action_t * op)
                                DBUS_TYPE_INVALID);
     }
 
-    if(op) {
+    if(op && path) {
```

Comment:
I don't follow how this returns PCMK_OCF_UNKNOWN_ERROR for soft errors.
If path is NULL because pcmk_dbus_type_check() returned false, or dbus_message_get_args() didn't set it, how will any error be reported?